src/components/Display.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+
+const display = props => {
+  const { result } = props;
+  return (
+    <div>
+      {result}
+    </div>
+  );
+};
+
+display.defaultProps = {
+  result: 0,
+};
```

Comment:
lint 14 should probably be 
``` 
result: '0'
```
since it's a string

Reply:
Thanks a lot! I will bring the required change!